mirgecom/boundary.py

```diff
@@ -138,4 +165,28 @@ def boundary_pair(self, discr, q, btag, **kwargs):
                                     momentum=ext_mom,
                                     species_mass=int_cv.species_mass)
 
-        return TracePair(btag, interior=int_soln, exterior=bndry_soln)
+        return bndry_soln
+
+    def exterior_grad_q(self, discr, grad_q, btag, **kwargs):
```

Comment:
It'd be nice if this could make use of @MTCam's upcoming BC work that's usable for both diffusion and fluid PDEs.